### PR TITLE
Generate data for graph option and update example

### PR DESCRIPTION
**Pull request by JasperCraeghs**
Both display options (`:col:` and `:graph:`) option are optional. They use the same filter options and, thus, the same defects. If both display options are not specified, the behaviour of the plugin stays the same, i.e. the default columns are used to generate the defects table.

### CODE REVIEW

File: mlx/coverity.py [hunk 1]
```diff
@@ -118,22 +117,24 @@ def run(self):
         # Process ``col`` option
         if 'col' in self.options:
             item_list_node['col'] = self.options['col'].split(',')
-        else:
+        elif 'chart' not in self.options:
             item_list_node['col'] = 'CID,Classification,Action,Comment'.split(',')
+        else:
+            item_list_node['col'] = []  # don't display a table if the ``chart`` option is present
```
Comment (Letme) [inline mlx/coverity.py]: I would say there are cases we want table plus graph so this should be adjusted.

**Review by Letme — CHANGES_REQUESTED** [review]

File: mlx/coverity.py (hunk 1 above)
Comment (JasperCraeghs) [reply mlx/coverity.py]: That is possible. Look at the if-part of the if-elif-else

Comment (Letme) [reply mlx/coverity.py]: I meant comment should be adjusted.

Comment (JasperCraeghs) [reply mlx/coverity.py]: 7416ca995e0732eececa5dcdb22cb438213f3200

**Review by Letme — APPROVED** [review]

**Pull request merged**